x-pack/plugins/secops/public/components/page/network/network_top_n_flow_table/translations.ts

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const SOURCE = i18n.translate('xpack.secops.networkTopNFlowTable.sourceTitle', {
+  defaultMessage: 'Top sources',
+});
+
+export const DESTINATION = i18n.translate('xpack.secops.networkTopNFlowTable.destinationTitle', {
+  defaultMessage: 'Top destinations',
+});
+
+export const SOURCE_IP = i18n.translate('xpack.secops.networkTopNFlowTable.topSourceIp', {
+  defaultMessage: 'Source Ip',
+});
+
+export const DESTINATION_IP = i18n.translate('xpack.secops.networkTopNFlowTable.topDestinationIp', {
+  defaultMessage: 'Destination Ip',
+});
+
+export const DOMAIN = i18n.translate('xpack.secops.networkTopNFlowTable.domain', {
+  defaultMessage: 'Domain',
+});
+
+export const BYTES = i18n.translate('xpack.secops.networkTopNFlowTable.bytes', {
+  defaultMessage: 'Bytes',
+});
```

Comment:
From the `i18n` docs example should these keys end with the word "Description"?
https://github.com/elastic/kibana/blob/master/packages/kbn-i18n/GUIDELINE.md#message-types
```
kbn.management.editIndexPattern.scripted.table.nameDescription
```